lib/logstash/inputs/gelf.rb

```diff
@@ -96,7 +96,12 @@ def udp_listener(output_queue)
       # Gelfd parser outputs null if it received and cached a non-final chunk
       next if data.nil?
 
-      event = LogStash::Event.new(LogStash::Json.load(data))
+      begin
+        event = LogStash::Event.new(LogStash::Json.load(data))
+      rescue => ex
+        @logger.warn("JSON failed to parse a message. Skipping", :exception => ex, :backtrace => ex.backtrace)
```

Comment:
I like the idea, sounds better than simply dropping the message. I can make the change in this PR. Should I also add the `rescue StandardError` bits?